src/main/java/com/mailjet/client/transactional/response/MessageResult.java

```diff
@@ -0,0 +1,44 @@
+package com.mailjet.client.transactional.response;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MessageResult {
+
+    /**
+     * Indicates the sending status of the message.
+     * Possible values:
+     * success
+     * error
+     */
+    private String status;
+
+    /**
+     * List containing information about any errors with the processing of this message.
+     * Will be displayed only if processing errors occur.
+     * Each error will generate a separate error object with the below properties.
+     */
+    private SendEmailError[] errors;
```

Comment:
why is this an array instead of a list?

Reply:
I feel like here array is more semantically suitable - it is the response model, so it shouldn't be modified after creatation.